draft-ietf-rtcweb-jsep.xml

```diff
@@ -1577,7 +1577,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             The isStopped method returns "true" if
             the transceiver has been stopped, either by a call
             to stopTransceiver or by a call to
-            setLocalDescription, and "false" otherwise.
+            setRemoteDescription, and "false" otherwise.
```

Comment:
I think there's a problem here.   An offer can't stop a transceiver, because it's possible that will be rolled back.  An answer must be the thing that can stop a transceiver.   And that would only happen if the remote side offered to stop the transceiver, which means the answer that actually stops it (in a way that can't be rolled back) is the setLocalDescription(answer), not the setRemoteDescription(offer).